### ISSUE TITLE
Add offset to dialog relative to the center bottom position of the screen

### ISSUE THREAD

**Issue by coppolaemilio** [coppolaemilio/dialogic#272]

**Comment by Jowan-Spooner** [comment]
Maybe it would be cool to have an option to pin it to other places (make it relative to them) too? Becaus that would mean more freedom for theplacement of the dialogue box.
 I got the idea from the speech bubble that AnidemDex posted inthe Discord server.

**Comment by coppolaemilio** [comment]
Yeah, I was thinking of a selector for docking it to top, left, bottom, right and the offset. But for now I wanted to make a note about making this one since I think it should be simple enough
